### PR TITLE
test: move relevant test from createImageWizard to targetEnvironment test file

### PR DESCRIPTION
**Pull request by mgold1234**
this commit move relvant test from creatreImageQizrad to targetEnvironment test file

### CODE REVIEW

**Comment by regexowl** [comment]
Can you please fix the typos in the name and description? Also let's "sort' this under tests to make the component part of the title clearer `test: move relevant test from createImageWizard to targetEnvironment test file`

File: src/test/Components/CreateImageWizard/CreateImageWizard.test.tsx [hunk 1]
```diff
@@ -88,419 +82,6 @@ describe('Create Image Wizard', () => {
...
-    );
-    expect(await getNextButton()).toHaveClass('pf-m-disabled');
-    expect(await getNextButton()).toBeDisabled();
-    await waitFor(async () =>
-      user.type(await screen.findByTestId('principal'), 'test@test.com')
-    );
-    expect(await getNextButton()).not.toHaveClass('pf-m-disabled');
-    expect(await getNextButton()).toBeEnabled();
-  });
-});
```
Comment (regexowl) [inline src/test/Components/CreateImageWizard/CreateImageWizard.test.tsx:502]: And this to the GCP test file? `src/test/Components/CreateImageWizard/steps/TargetEnvironment/GCPTarget.test.tsx`

**Review by regexowl — APPROVED** [review]
Yay! Cleaning up! :broom:

**Comment by mgold1234** [comment]
sorry @regexowl I added another last changes :)

File: src/test/Components/CreateImageWizard/steps/TargetEnvironment/AwsTarget.test.tsx [hunk 2]
```diff
@@ -83,6 +99,181 @@ const enterAccountId = async () => {
   await waitFor(async () => user.type(awsAccountIdTextbox, '123123123123'));
 };
 
+let router: RemixRouter | undefined = undefined;
+const routes = [
+  {
+    path: 'insights/image-builder/imagewizard/:composeId?',
+    element: <CreateImageWizard />,
+  },
+];
```
Comment (regexowl) [inline src/test/Components/CreateImageWizard/steps/TargetEnvironment/AwsTarget.test.tsx:108]: Ah, I see. There's a default path to `/insights/image-builder` missing in the routes. Adding it to the AWS and GCP test suite should do the trick :magic_wand: 

```suggestion
const routes = [
  {
    path: 'insights/image-builder/*',
    element: <div />,
  },
  {
    path: 'insights/image-builder/imagewizard/:composeId?',
    element: <CreateImageWizard />,
  },
];
```

**Review by regexowl — APPROVED** [review]
Thank you!

**Pull request merged**